generators/voip.html.sh

```diff
@@ -28,6 +28,8 @@ Packages that operate on voip programs and protocols.</p>
                 <br />
                 <p>
                     <b>Tool count:</b> <a href="">${VOIP_COUNT}</a>
+                    <input type="text" id="searchTools" onkeyup="searchTools()" placeholder="Input tool name" title="Type in a name">
```

Comment:
double lines here and also check below and webapp.html.sh :)

Reply:
Okay, will check